src/components/Main/Schedule/NewScheduleForm.tsx

```diff
@@ -0,0 +1,159 @@
+    view: string,
+    team: number,
+  ) => void;
+  onClose: () => void;
+}
+
+export const NewScheduleForm = ({ onSubmit, onClose }: NewScheduleFormProps) => {
+  const [scheduleTitle, setScheduleName] = useState('');
+  const [startDate, setStartDate] = useState(toDateString(new Date()));
+  const [endDate, setEndDate] = useState(toDateString(new Date()));
+  // const [view, setView] = useState<string>('weekly');
+  const view = 'weekly';
+  const [team, setTeam] = useState<number>(0);
+  const classes = useStyles();
+
+  //needed to format date so that the date picker can display it properly
+  function toDateString(date: Date): string {
+    return `${date.getFullYear()}-${date.getMonth()}-${date.getDate()}`;
+  }
+
+  return (
+    <div className={classes.root}>
+      New Schedule Form
+      <form className={classes.formStyle}>
+        <div className={classes.tooltipContainer}>
+          <FormField
+            className={classes.nameInput}
+            name="Name"
+            label="Schedule Name"
+            value={scheduleTitle}
+            handleChange={setScheduleName}
+          />
+          <Tooltip
+            id="scheduleName"
+            text="Example name: Jan-Mar Schedule. Must be unique"
+          />
+        </div>
+        <div className={classes.tooltipContainer}>
+          <TextField
+            className={classes.datePicker}
+            variant="outlined"
+            id="start date"
+            label="Start Date"
+            type="date"
+            InputLabelProps={{
+              shrink: true,
+            }}
+            value={startDate}
+            onChange={(e) => setStartDate(e.target.value)}
+          />
+          <TextField
+            className={classes.datePicker}
+            variant="outlined"
+            id="end date"
+            label="End Date"
+            type="date"
+            InputLabelProps={{
+              shrink: true,
+            }}
+            value={endDate}
+            onChange={(e) => setEndDate(e.target.value)}
+          />
+          <Tooltip
+            id="datePicker"
+            text="Select the begin date and end date for this schedule"
+          />
+        </div>
+
+        <FormControl className={classes.selectContainer}>
+          {/* <InputLabel>Team</InputLabel> */}
+          <Select
+            className={classes.selectInput}
+            value={team}
+            required={true}
+            variant="outlined"
+            onChange={(e: React.ChangeEvent<{ name: string; value: number }>) =>
+              setTeam(e.target.value)
+            }
+          >
+            <MenuItem value={0}>Church Council</MenuItem>
+            <MenuItem value={1}>RE</MenuItem>
+          </Select>
+          <FormHelperText>Team</FormHelperText>
+          <Tooltip id="Team" text="Select who is able to edit this schedule" />
+        </FormControl>
+      </form>
+      <button onClick={() => onSubmit(scheduleTitle, startDate, endDate, view, team)}>
```

Comment:
this function should only fire if all validations are met.
1) Title shouldn't be longer than 32 characters
2) all fields must be filled and not null/empty